functions/update-dynamodb/function.yml

```diff
@@ -0,0 +1,4 @@
+handler: handler.handle
```

Comment:
1. The path is wrong. Please include the correct path of the handler.
2. Could you also specify 23 seconds timeout of this lambda function?
3. You are missing the top key for this yaml file. At the top you should specify the lambda name - something like `update-dynamodb:` and then put all other yaml properties under this name